wiki/PDBParser.md

```diff
@@ -54,43 +54,57 @@ import sys, os, warnings
 def parse_structure(path):
     """ Parses a PDB file """
 
-    s = P.get_structure('test', path)
+    s = P.get_structure("test", path)
 
     return 0
 
+
 def fancy_output(tps):
     """ Outputs the results in a nicer way """
 
     print("# Bio.PDB PDBParser Benchmark")
     print()
     print("Structure \tLenght \tTime Spent (ms)")
-    for i,s in enumerate(tps):
-        print(" %s\t (%s) \t%3.3f" % (os.path.basename(pdb_library[i]),
-                                      pdb_length[i], s))
+    for i, s in enumerate(tps):
+        print(
+            " %s\t (%s) \t%3.3f" % (os.path.basename(pdb_library[i]), pdb_length[i], s)
+        )
     print()
-    print("Total time spent: %5.3fs" % (sum(tps)/1000))
-    print("Average time per structure: %5.3fms" % (sum(tps)/len(tps)))
+    print("Total time spent: %5.3fs" % (sum(tps) / 1000))
+    print("Average time per structure: %5.3fms" % (sum(tps) / len(tps)))
+
 
-if __name__=='__main__':
+if __name__ == "__main__":
 
     import time, gc
     from Bio.PDB import PDBParser
-    P = PDBParser(PERMISSIVE=1) # For the pdb_enhancements branch benchmarking, PERMISSIVE was set to 2 (silence warnings).
+
+    P = PDBParser(
+        PERMISSIVE=1
+    )  # For the pdb_enhancements branch benchmarking, PERMISSIVE was set to 2 (silence warnings).
```

Comment:
Do we have `PERMISSIVE=2`? Isn't that `QUIET=1`?

Reply:
That may be obsolete / out of date. I didn't check.

Reply:
I downloaded the entire PDB (both in PDB and mmCIF format) overnight and I'm running a similar benchmarking script. Thought it could use an update. I'll update the page when it's done. The idea is to time the parsing/writing and do two comparisons: round-trip (parse, write, parse, compare) and pdb-vs-mmcif (when possible). Should be a good consistency check.